spec/models/trademark_decision_spec.rb

```diff
@@ -1,23 +0,0 @@
-require "spec_helper"
-require "models/valid_against_schema"
-
-RSpec.describe TrademarkDecision do
-  let(:payload) { FactoryBot.create(:trademark_decision) }
-  include_examples "it saves payloads that are valid against the 'specialist_document' schema"
-  subject(:trademark_decision) { described_class.from_publishing_api(payload) }
-
-  it "is not exportable" do
-    expect(described_class).not_to be_exportable
-  end
-
-  describe "validations" do
-    it "is valid from the payload" do
-      expect(trademark_decision).to be_valid
-    end
-
-    it "is invalid if the organisation is missing" do
```

Comment:
Agree this sort of thing should go - but we should probably have a generic test that a facet marked as 'required' must have a value specified.